### ISSUE TITLE
why stonger baseline not using bnnneck for triplet-loss

### ISSUE THREAD

**Issue by liuguochao** [JDAI-CV/fast-reid#369]
if self.neck_feat == "before":  feat = global_feat[..., 0, 0]
  elif self.neck_feat == "after": feat = bn_feat
  else:                           raise KeyError(f"{self.neck_feat} is invalid for MODEL.HEADS.NECK_FEAT")

  return {
      "cls_outputs": cls_outputs,
      "pred_class_logits": pred_class_logits,
      "features": feat,
  }

**Comment by L1aoXingyu** [comment]
I found it made no differences when using the feature before BN or after BN.
